blocks/single-chain-block/chains/single-chain/default.jsx

```diff
@@ -1,21 +1,23 @@
-import React from "react";
+import React, { Fragment } from "react";
 import PropTypes from "@arc-fusion/prop-types";
-import { Heading, HeadingSection } from "@wpmedia/shared-styles";
+import { Heading, HeadingSection, Stack } from "@wpmedia/arc-themes-components";
 
-import "@wpmedia/shared-styles/scss/_chains.scss";
+const BLOCK_CLASS_NAME = "b-single-chain";
 
 const SingleChain = ({ children, customFields = {} }) => {
 	const { heading = null } = customFields;
 
-	if (!heading) {
-		return <>{children}</>;
-	}
+	const Wrapper = heading ? HeadingSection : Fragment;
```

Comment:
Perhaps to fulfill the null test below you commented out:

```
if (!heading && React.children(children).length === 0) { return null }
```

Reply:
Thanks @nschubach ...geesh, went to work on the other chain tests and forgot to come back to this. I put in something close to your suggestion `Children.count(children) ` and reinstated the tests.